# Voice Command Message
std_msgs/Header header

# Voice command text
string command_text
